checker/client.go

```diff
@@ -45,7 +45,7 @@ func GetClients(ctx context.Context, repoURI, localURI string, logger *log.Logge
 			localdir.CreateLocalDirClient(ctx, logger), /*repoClient*/
 			nil, /*ossFuzzClient*/
 			nil, /*ciiClient*/
-			nil, /*vulnClient*/
+			clients.DefaultVulnerabilitiesClient(), /*vulnClient*/
```

Comment:
This is because I added the Vulnerabilities check to support FileBased as well as the original CommitBased. 
https://github.com/ossf/scorecard/blob/ca78e15ab8e75806e941104c049b517e21af4a81/checks/vulnerabilities.go#L29-L32
Since now we can scan manifests as well as the git commit hash.